lib/config.js

```diff
@@ -512,9 +516,24 @@ function normalizeBytes (opts) {
   }
 }
 
+function normalizePositiveTime (opts, logger) {
```

Comment:
nit: Would it be worth getting some unit tests around both `normalizePositiveTime` and `toSeconds`?  The case for this: They both seem like classic candidates for these sorts of tests (clear inputs, clear outputs) and having the edge-cases/invalid configurations in tests makes the intended behavior of these functions clearer in the future.

Reply:
I pale a little bit at the warty test/config.test.js ... :)  but I'll give it a go.

Reply:
Added in https://github.com/elastic/apm-agent-nodejs/pull/2330/commits/2160e98ac0a1d4eca8d7191180804eed143b7e5a